test.sh

```diff
@@ -352,24 +380,30 @@ function build-and-run-sample-c-appln-with-LOC-encoding()
 # Driver method to run through all variations of client-server perf tests
 #
 # Parameters:
-#   $1  - (Opt) # of messages to exchange from client -> server (default: 1000)
-#   $2  - (Opt) # of clients to start-up (default: 5)
+#   $1  - (Opt) Arg to select clock-ID to use
+#   $2  - (Opt) # of messages to exchange from client -> server (default: 1000)
+#   $3  - (Opt) # of clients to start-up (default: 5)
 #
 # Usage:
-#  $ ./test.sh run-all-client-server-perf-tests $((100 * 1000)) > /tmp/perf-test.100K-rows.5ct.out 2>&1
+#  $ ./test.sh run-all-client-server-perf-tests --clock-default $((100 * 1000)) > /tmp/perf-test.100K-rows.5ct.out 2>&1
 #
 #  $ egrep -B1 -E 'Start Server|, num_ops=|Unpacked nentries|throughput=' /tmp/perf-test.100K-rows.5ct.out | egrep -v -E 'Client-throughput|enabled on server-side.'
 
 # #############################################################################
 function run-all-client-server-perf-tests()
 {
-    local num_msgs_per_client=1000
+    # Reset global if arg-1 provided. Minions will grab it from there.
     if [ $# -ge 1 ]; then
-        num_msgs_per_client=$1
+        SvrClockArg=$1
```

Comment:
Minor rearrangement of args to this method to simplify common-case execution of `test.sh` with another clock.

E.g. `$ ./test.sh run-all-client-server-perf-tests --clock-process-cputime-id`

to run a small workload consisting of 5-clients and 1000 msgts, with above run-parameters.